### PR TITLE
Update README.md | Added instruction for Debian/Ubuntu

### PR DESCRIPTION
**Pull request by T-Blitz**
Added the Debian/Ubuntu way to install and added a hyper link to the gnu 'gettext' wiki.

### CODE REVIEW

**Comment by T-Blitz** [comment]
okay, no idea why the review failed

File: README.md [hunk 1]
```diff
@@ -98,9 +98,17 @@ System-wide install:
 ```bash
 pip install nautilus-open-any-terminal
 ```
+### For Debian/Ubuntu based Distros.
 
+Download the .dep package, for either Nautilus and or Caja from the [GitHub releases page](https://github.com/Stunkymonkey/nautilus-open-any-terminal/releases/latest).
```
[the remark this answers is not in the document]
Comment (T-Blitz) [reply README.md:103]: pain

File: README.md [hunk 2]
```diff
@@ -98,9 +98,17 @@ System-wide install:
...
+After that install it either with the apt command:
+
+```bash
+dpkg -i <package_name>.deb
```
[the remark this answers is not in the document]
Comment (T-Blitz) [reply README.md]: okay, made all 3 changes

**Comment by T-Blitz** [comment]
updated with typo fixes and command recommendation from lvxnull2

File: README.md [hunk 3]
```diff
@@ -98,9 +98,17 @@ System-wide install:
 ```bash
 pip install nautilus-open-any-terminal
 ```
+### For Debian/Ubuntu based Distros.
```
Comment (Stunkymonkey) [inline README.md:101]: to fix markdown linter:
```suggestion
### For Debian/Ubuntu based Distros
```

File: README.md [hunk 4]
```diff
@@ -98,9 +98,17 @@ System-wide install:
...
 
+Download the .deb package, for either Nautilus and or Caja from the [GitHub releases page](https://github.com/Stunkymonkey/nautilus-open-any-terminal/releases/latest).
+After that install it either with the apt command:
+
+```bash
+apt install <package_name>.deb
+```
+or depending on your Linux-Distro, you can just double click the .deb file and install it that way.
```
Comment (Stunkymonkey) [inline README.md:109]: fix natural-language:
```suggestion
or depending on your Linux-Distro, you can just double-click the .deb file and install it that way.
```

**Comment by T-Blitz** [comment]
Please pardon my lint based skill issue. But what am i missing....?

**Comment by Stunkymonkey** [comment]
@T-Blitz from the CI: `    109:49  ✓ error  Incorrect term: “double click”, use “double-click” instead  terminology`

**Comment by T-Blitz** [comment]
oh... i should go to bed soon, i thought it was the other way around 
pardon me

**Review by lvxnull2 — APPROVED** [review]

**Pull request merged**